source/globalCommands.py

```diff
@@ -3315,6 +3316,45 @@ def script_braille_cycleShowSelection(self, gesture: inputCore.InputGesture) ->
 			msg = _("Braille show selection %s") % BoolFlag[nextName].displayString
 		ui.message(msg)
 
+	@script(
+		# Translators: Input help mode message for cycle through
+		# braille route review cursor and system caret command.
+		description=_("Cycle through the braille route review cursor and system caret states"),
+		category=SCRCAT_BRAILLE
+	)
+	def script_braille_cycleRouteReviewCursorAndSystemCaret(self, gesture: inputCore.InputGesture) -> None:
+		"""If braille is not tethered to focus, set next state of braille route review cursor and system caret.

```

Comment:
Note that this doc string gets lost after decoration. Better make this generic comment style.